src/templates/RegionPage.tsx

```diff
@@ -32,6 +39,11 @@ const RegionPageTemplate: FunctionComponent<Props> = ({ data }) => {
         image={region.mapPhoto?.gatsbyImageData}
         alt="Map of {region.name}"
       />
+      <h1 className="text-xl">To Shipments</h1>
+      <p>{JSON.stringify(toShipments)}</p>
+      <h1 className="text-xl">From Shipments</h1>
+      <p>{JSON.stringify(fromShipments)}</p>
+
```

Comment:
Not a huge deal, but using this way, if a shipment is from the region to the same region, it will show up in both lists here.